src/test/java/de/learnlib/ralib/learning/rattt/LearnSipIOTest.java

```diff
@@ -98,7 +98,7 @@ public void learnSipIO() {
         TreeOracleFactory hypFactory = (RegisterAutomaton hyp) ->
                 new MultiTheoryTreeOracle(new SimulatorOracle(hyp), teachers, consts, solver);
 
-        RaTTT rattt = new RaTTT(mto, hypFactory, mlo, consts, true, actions);
+        RaLambda rattt = new RaLambda(mto, hypFactory, mlo, consts, true, actions);
```

Comment:
Even though a commit in this branch renamed `RaTTT` to `RaLambda` in various files, there are still occurrences of `rattt`-named variables both in the code (e.g. in this file)  and in directories (e.g. in tests).  I think we should also rename these.

Reply:
Addressed by #37